### DISCUSSION TITLE
账号切换同时会将另一个账号《Wallpaper Engine:壁纸引擎》所创意工坊订购的作品一同下载

### DISCUSSION THREAD

**Discussion by sos932207069sos** [BeyondDimension/SteamTools#2462] [General]
在整理壁纸引擎这款开车软件，居然发现其他我没有订购我没有的作品。我不知道这是那个人遗漏下来的，但这很占用我的内存空间，之后我一键登录了我朋友的账号发现这些作品是他订购的而且不符合我的xp,我想删除他们,但是Wallpaper Engine的文件非常乱!我和我朋友的作品混杂在了一起，我删除后还会重新下载，这让我陷入了迷茫

**Comment by pjy612** [comment]
你用了你朋友的号，还想把别人的订阅给关掉？ 乐... 
这个算是 steam 默认的机制吧，要改的话  就要去区分和自定义哪些工坊要下载哪些不下载。
之后不确定你是否又会觉得切到自己的号 又不能全部下载。。。

**Reply by sos932207069sos** [reply]
不好意思我可能没说清楚题目，我在登陆自己的号，同时订购的是我朋友号上的作品加上我的作品

**Reply by sos932207069sos** [reply]
在Wallpaper Engine自动更新的时候，就会莫名其妙订购了，我朋友创意工坊里的物品，我就觉得很奇怪。Watt Toolkit的账号切换功能和这件事到底有没有什么关联？（而且你“乐...”个毛啊？大早上脑抽了？）

**Reply by sos932207069sos** [reply]
我说想删除它们是在根目录里面删除，不好意思我没说清楚，我没用过github，所以我不知道去哪里回复这件事情，难道github都是这种态度和人说话的吗？真TM的没素质

**Reply by sos932207069sos** [reply]
你以为你是谁敢在我评论区回复，能滚哪里去就滚哪里去!

**Reply by Mossimos** [reply]
> 你以为你是谁敢在我评论区回复，能滚哪里去就滚哪里去!

这里是讨论区请文明讨论
另外创意工坊 Steam 默认 就是下载全部订阅。

**Reply by pjy612** [reply]
steam订阅就是全部下载所有可能的订阅，就算你本地删掉后，后面一旦订阅更新，Steam还是会给你拉下来。
所以 等于你要不想订阅就得自己去取消。然后 你又用的是别人的号。。。所以就等于你要去动别人的订阅。
总之 推荐自己入正。这种工具类的 就别白嫖了。
steam的 创意工坊订阅 和 DLC不一样，DLC能主动选择是否勾选。创意工坊的配置 只有一个取消订阅。
so，这种费力讨好的事情... 推荐自己学习然后动手解决。。。

**Reply by pjy612** [reply]
另外创意工坊的机制也比较特别，其他游戏你可以设置成 启动时检查更新。
但是 创意工坊是否能 将同步也放在启动时 尚不清楚。
推荐是去让你朋友开家庭共享给你，或者借朋友号 离线玩指特定的几个游戏。这样控制权多少都在自己的号上。
要么就是直接给Steam提工单，让他们别一下子下载一堆订阅。

**Reply by sos932207069sos** [reply]
我自己有壁纸引擎登陆自己的账号时会下载多余的创意工坊内容，所以我感到很奇怪
而且并不会显示在软件中所以登陆我朋友的号去查看是否是这个原因，发现这些是我朋友订购的
steam条例不允许任何形式的租借账号,Steam提工单我不知道会不会导致朋友的号被封禁
这个建议可以让steam采纳一下，我不想提供什么工单真麻烦！我不白嫖还不行嘛
